crates/holochain_cascade/src/lib.rs

```diff
@@ -1,8 +1,22 @@
-//! # Cascade
+//! The Cascade is a multi-tiered accessor for Holochain DHT data.
+//!
+//! It is named "the Cascade" because it performs "cascading" gets across multiple sources:
+//! - First attempts to read the local storage
+//! - If that fails, attempt to read data from the network cache
+//! - If that fails, do a network request for the data, caching it if found
+//!
 //! ## Retrieve vs Get
-//! Get checks CRUD metadata before returning an the data
-//! where as retrieve only checks that where the data was found
-//! the appropriate validation has been run.
+//!
+//! There are two words used in cascade functions: "get", and "retrieve".
+//! They mean distinct things:
+//!
+//! - "get" checks CRUD metadata before returning the data, so for instance, Deletes
```

Comment:
Get only checks crud status for `dht_get`. I think the real difference is that it checks the validation status.